core/ui/src/components/applications-center/AddNoteModal.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <NsModal
+    :visible="visible"
+    :isLoading="loading"
+    @primary-click="handleSave"
+    @hide="handleHide"
+    size="default"
+  >
+    <template slot="title">{{
+      isEdit ? $t("applications.edit_note") : $t("applications.add_note")
+    }}</template>
+    <template slot="content">
+      <div class="mg-bottom-md">{{ $t("applications.note_description") }}</div>
+      <div class="add-note-modal-content">
```

Comment:
We can use tailwind-like classes defined in `_utils.scss`
```suggestion
      <div class="flex flex-col">
```